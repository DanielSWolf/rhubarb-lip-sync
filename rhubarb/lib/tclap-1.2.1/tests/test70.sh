#!/bin/sh

# Checks that parsing exceptions are properly
# propagated to the caller.
../examples/test18 --help > tmp.out 2>&1

if cmp -s tmp.out $srcdir/test70.out; then
	exit 0
else
	exit 1
fi

