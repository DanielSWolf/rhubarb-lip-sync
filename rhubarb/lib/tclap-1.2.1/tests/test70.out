
USAGE: 

   ../examples/test18  [--] [--version] [-h]


Where: 

   --,  --ignore_rest
     Ignores the rest of the labeled arguments following this flag.

   --version
     Displays version information and exits.

   -h,  --help
     Displays usage information and exits.


   Command description message

Exiting on ExitException.
